.bash_invenio

```diff
@@ -137,10 +145,16 @@ $text
 EOF
 
   # add release notes after =======
-  sed -i "/=======/r /tmp/release-notes.txt" CHANGES.rst
+  rel_file=$(find . -maxdepth 1 -type f  -name "CHANGES.*")
+  sed -i "/=======/r /tmp/release-notes.txt" $rel_file
 
-  # update __version__ with the new tag
-  sed -i "s/^__version__.*/__version__ = \"${tag_name:1}\"/" $init_file
+  if [ $npmpackage -gt 0 ]
```

Comment:
the same as above and yeah you could place the "package.json" in a variable called NPM_PACKAGE_FILE
`NPM_PACKAGE_FILE="package.json"`